src/interface/chessBoardInterface.js

```diff
@@ -1,5 +1,57 @@
-export default class ChessBoardInterface {
-    constructor(parentNode, onFieldClick) {
+// export default class ChessBoardInterface {
+//     constructor(parentNode, onFieldClick) {
 
-    }
-}
\ No newline at end of file
+//     }
+// }
+
+var symbols = {
+    "rookWhite" : "\u2656",
+    "knightWhite" : "\u2658",
+    "bishopWhite" : "\u2657",
+    "queenWhite" : "\u2655",
+    "kingWhite" : "\u2654",
+    "pawnWhite" : "\u2659",
+
+    "rookBlack" : "\u265c",
+    "knightBlack" : "\u265e",
+    "bishopBlack" : "\u265d",
+    "queenBlack" : "\u265b",
+    "kingBlack" : "\u265a",
+    "pawnBlack" : "\u265f",
+};
+
+function ChessBoard() {
+
+    var board = document.querySelector('#board');
+
+    var cells = [];
+
+    this.createBoard = function () {
+         board.style.width = '400px';
+         board.style.height = '400px';
+
+            for (var i = 1; i <= 8; i++) {
+                var divRow = document.createElement('div');
+                divRow.classList.add("row");
+                    for (var j = 1; j <= 8; j++) {
+                        var div = document.createElement('div');
+                        div.classList.add("unit");
+                        div.setAttribute("data-x", i);
+                        div.setAttribute("data-y", j);
+                        //div.innerText = symbols.bishopBlack;
+                
+                    if (i % 2 == j % 2) {
+                      div.style.backgroundColor = "white";
+                    } else {
+                      div.style.backgroundColor = "#7f7f7f";
+                    }
+                    divRow.append(div);
+                  }
+                  board.append(divRow);
+                }
+    };
+
+}
+
+var chess = new ChessBoard();
+chess.createBoard();
```

Comment:
Tego rodzaju kod powinien być wykonywany w ramach inicjalizacji aplikacji raczej niż być wpisanym wprost do pliku chessBoardInterface.js - zwłaszcza, że w ten sposób nie ma jak go sensownie wywołać.

Co więcej, przy takiej implementacji kłopotliwe staje się dopisywanie do obiektu kolejnych funkcji, czy to tych należących do początkowego zadania (podświetlanie pól) czy późniejszych (kiedy przyjdzie do wyświetlania figur). 

Co więcej, w tej chwili nic z tego pliku nie eksportujesz. Jeśli wziąć ten kod i przekleić do index.js to będzie działać, ale jest problem z tym, by go sensownie zaimportować. Zwróć proszę uwagę na oryginalną zawartość tego pliku - to byłby najlepszy sposób, żeby się za to zabrać. **Stworzyć (i eksportować) klasę ChessBoardInterface, która przyjmuje jako argument konstruktora węzeł, do którego ma się appendować.** Jeśli zrobi się to w ten sposób, to wtedy w odpowiednim miejscu (na razie w index.js) będzie można tę klasę zaimportować, stworzyć pochodny po niej obiekt (podając mu, gdzie ma się umieścić, zamiast pozwalać mu, żeby sam sobie miejsca szukał) i wygodnie się z nim komunikować.

Reply:
in progress

Reply:
changed function to class, but also in progress

Reply:
I added some new code with constructor